Omise.Examples/ExampleInfo.cs

```diff
@@ -1,36 +1,35 @@
-﻿
-namespace Omise.Examples
+﻿namespace Omise.Examples
 {
     public class ExampleInfo
     {
-        public const string OMISE_PKEY = "pkey_test_5alr4tca4bpu0cirhck";
-        public const string OMISE_SKEY = "skey_test_5alr4tz3so072nhkipj";
+        public const string OMISE_PKEY = "pkey_test_5uhkt2eb9s1xtbbp4v1";
+        public const string OMISE_SKEY = "skey_test_5uhkt2fyyjdhork2sjh";

```

Comment:
Can you please help me understand why do we need active keys here?

Reply:
Because the example app uses real api calls although I would say its better to indicate to the user that he should replace these parameters with his own parameters.

Reply:
Can we handle this via env variables with the help of library like [dotenv.net](https://github.com/bolorundurowb/dotenv.net)? So that we don't have to put the real keys here.